src/cascadia/TerminalApp/TerminalTab.h

```diff
@@ -21,7 +21,6 @@ namespace winrt::TerminalApp::implementation
     struct TerminalTab : TerminalTabT<TerminalTab, TabBase>
     {
     public:
-        TerminalTab(const winrt::Microsoft::Terminal::Settings::Model::Profile& profile, const winrt::Microsoft::Terminal::Control::TermControl& control);

```

Comment:
Oh man. This is hardcore gonna conflict with @zadjii-msft's warning dialog PR. I'll come back to this once that merges.